### ISSUE TITLE
Internal Server Error via API on Available Orders Search in Existing but Closed Marketplace

### ISSUE THREAD

**Issue by darkfishy** [flexemarkets/fm-issues#26]
Steps to reproduce:
1. Create a new marketplace with one market
2. Open the marketplace
3. Execute at least one trade in a way that there are no pending orders
  a. Send a `SELL` order
  b. Send an equivalent `BUY` order
  c. Trade takes place
3. Close the marketplace
4. `GET` the `URL('https://fm-data.herokuapp.com/api/orders/search/orders-available?marketId={market_id}')` where `market_id` is for the market created above.

Response received: 
```
{
  "status" : "INTERNAL_SERVER_ERROR",
  "error" : "SERVER_ERROR",
  "message" : "Snap! :o"
}
```

Expected response:
```
{
  "_embedded": {
    "orders": []
  },
  "_links": {
    "self": {
      "href": "https://fm-data.herokuapp.com/api/orders/search/orders-available?marketId={market_id}&page=0&size=1000"
    }
  },
  "page": {
    "size": 1000,
    "totalElements": 0,
    "totalPages": 1,
    "number": 0
  }
}
```

**Comment by JanStureNielsen** [comment]
I'm not able to reproduce this, @darkfishy, on AHM -- would you try? If you're able to reproduce, add the curl command to the issue and re-open.

**Comment by darkfishy** [comment]
@JanStureNielsen I cannot reproduce this either. I have a suspicion that it got resolved after issue #24
